apps/framework-cli/src/cli/routines/metrics_console/run_console/app.rs

```diff
@@ -68,45 +55,88 @@ pub struct App {
     pub streaming_functions_bytes_per_sec: HashMap<String, u64>,
 }
 
+pub struct AppOverviewMetrics {
+    pub average: f64,
+    pub total_requests: f64,
+    pub requests_per_sec: f64,
+    pub summary: Vec<PathMetricsData>,
+    pub main_bytes_data: MainBytesParsedData,
+}
+
+pub struct AppTableScrollData {
+    pub endpoint_starting_row: usize,
+    pub kafka_starting_row: usize,
+    pub streaming_functions_table_starting_row: usize,
+}
+
+pub struct AppPathsData {
+    pub path_detailed_data: Option<Vec<HistogramCount>>,
+    pub path_requests_per_sec: HashMap<String, f64>,
+    pub requests_per_sec_vec: HashMap<String, Vec<u64>>,
+    pub parsed_bytes_data: PathBytesParsedData,
+}
+
+pub struct App {
+    pub table_state: TableState,
+    pub viewport: Rect,
+    pub state: State,
+    pub running: bool,
+    pub table_scroll_data: AppTableScrollData,
+    pub paths_data: AppPathsData,
+    pub kafka_metrics: AppKafkaMetrics,
+    pub streaming_functions_metrics: AppStreamingFunctionsMetrics,
+    pub overview_data: AppOverviewMetrics,
+}
+
 impl Default for App {
     fn default() -> Self {
         Self {
             table_state: TableState::Endpoint,
-            viewport: Rect::new(0, 0, 0, 0),
+            viewport: Rect::default(),
             state: State::Main(),
             running: true,
-            average: 0.0,
-            requests_per_sec: 0.0,
-            total_requests: 0.0,
-            summary: vec![],
-            endpoint_starting_row: 0,
-            kafka_starting_row: 0,
-            streaming_functions_table_starting_row: 0,
-            path_detailed_data: vec![].into(),
-            path_requests_per_sec: HashMap::new(),
-            requests_per_sec_vec: HashMap::new(),
-            parsed_bytes_data: PathBytesParsedData {
-                previous_bytes: HashMap::new(),
-                path_bytes_in_per_sec_vec: HashMap::new(),
-                path_bytes_out_per_sec_vec: HashMap::new(),
+            table_scroll_data: AppTableScrollData {
+                endpoint_starting_row: 0,
+                kafka_starting_row: 0,
+                streaming_functions_table_starting_row: 0,
+            },
+            paths_data: AppPathsData {
+                path_detailed_data: None,
+                path_requests_per_sec: HashMap::new(),
+                requests_per_sec_vec: HashMap::new(),
+                parsed_bytes_data: PathBytesParsedData {
+                    previous_bytes: HashMap::new(),
+                    path_bytes_in_per_sec_vec: HashMap::new(),
+                    path_bytes_out_per_sec_vec: HashMap::new(),
+                },
+            },
+            kafka_metrics: AppKafkaMetrics {
```

Comment:
Those are the sync to clickhouse metrics right?

if so - would be great to have that explicit